lab-said/__test__/lib/.test.env

```diff
@@ -0,0 +1,3 @@
+MONGODB_URI = 'mongodb://localhost/cf-gram-div'
+APP_URI='localhost/4444'
```

Comment:
What is APP_URI? I couldn't find a reference to it in any other file. I think you wan this to be APP_SECRET.